.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python:
+  - "3.5"
+
+# command to install dependencies
+install: "pip install -r requirements.txt"
+
```

Comment:
I am not sure if we need to run something here to check the db as in the repo:
before_script: redis-cli ping
??